src/networkcreator.js

```diff
@@ -1008,7 +1010,7 @@ class NetworkCreator extends EventEmitter {
 
   generateGethService (volumes) {
     let gethService = {
-      image: 'livepeer/geth-with-livepeer-protocol:pm',
+      image: 'livepeer/geth-with-livepeer-protocol:pm-v1.7.3',
```

Comment:
Can we merge livepeer/docker-livepeer#10, push the updated image to DockerHub and then just use the latest version of the image here?

Reply:
I wanted to do that in the PR for changing the controller address so that it's clear that the new address belongs to the new image, but I can do it now.